casbin/internal_enforcer.py

```diff
@@ -37,6 +37,42 @@ def _add_policies(self,sec,ptype,rules):
                 self.watcher.update()
 
         return rules_added
+
+    def _update_policy(self, sec, ptype, old_rule, new_rule):
+        """updates a rule from the current policy."""
+        rule_updated = self.model.update_policy(sec, ptype, old_rule, new_rule)
+        if not rule_updated:
+            return rule_updated
+
+        if self.adapter and self.auto_save:
+            if hasattr(self.adapter, 'update_policy') is False:
+                return False
+
+            if self.adapter.update_policy(sec, ptype, old_rule, new_rule) is False:
```

Comment:
The adapter should implement `update_policy` and `update_policies`

Reply:
Do you mean there is no need to check the adaptar?